React/Base/RCTModuleMethod.mm

```diff
@@ -90,8 +90,8 @@ static BOOL RCTParseSelectorPart(const char **input, NSMutableString *selector)
 
 static BOOL RCTParseUnused(const char **input)
 {
-  return RCTReadString(input, "__unused") ||
-         RCTReadString(input, "__attribute__((unused))");
+  return RCTReadString(input, "__attribute__((unused))") ||
+         RCTReadString(input, "__unused");
```

Comment:
Actually `__unused` may not happen, because `__unused` macro would be expanded to `__attribute__((unused))` by compiler automatically.